packages/grid/x-data-grid-premium/src/components/GridFooterCell.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { getDataGridUtilityClass, GridRenderCellParams } from '@mui/x-data-grid';
+import { styled, Theme } from '@mui/material/styles';
+import { SxProps } from '@mui/system';
+import Box from '@mui/material/Box';
+import { unstable_composeClasses as composeClasses } from '@mui/material';
+import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+import { DataGridPremiumProcessedProps } from '../models/dataGridPremiumProps';
+
+const GridAggregationCellRoot = styled(Box, {
+  name: 'MuiDataGrid',
+  slot: 'AggregationCell',
+  overridesResolver: (_, styles) => styles.footerCell,
```

Comment:
The slot name is not matching the overrides key. `footerCell` is very wide, maybe `aggregationFooterCell`?

Reply:
No I just forgot to rename it when I renamed the component
Fixed